api/README.md

```diff
@@ -0,0 +1,158 @@
+### Summary
+
+This example shows how, having a screen, attach to it a newly created asset with js injection.
+
+For this, you would need
+ - an API token, which can be found on the web dashboard /manage/account/team.
+ - a paired screen id, which can be found either from the API or retrieved from the web dashboard screen URL:
+
+https://yourteam.screenlyapp.com/manage/screens/01FAZMW8E0CB2AM78N63QP406Y
```

Comment:
We should use the API for this. We don't want people to have to manually parse URLs to dig out data.